### ISSUE TITLE
Contribute to typeshed

### ISSUE THREAD

**Issue by bellini666** [sbdchd/django-types#38]
Have you thought about contributing this project to https://github.com/python/typeshed?

Unlike `django-stubs`, which has a hard dependency on mypy, this project is generic enough to be included there. Doing that would bring more attention to these typings.

**Comment by sbdchd** [comment]
yeah I'm open to it, but the main issue is that these types are somewhat opinionated as I tend towards making things more strict even if then runtime types are more flexible
